Bottom: 9e8231ab5e3a208fb3f77784cd232e5b1734f0fc
Top:    d6530e1c569a3ed11b2f0cd3c27bc26fae887dc9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 15:47:25 +0000

Refresh of continue-tinkering-with-3d

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 9341618..0197118 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -613,13 +613,3 @@ object3d_test_board_outline (void)
 
   g_list_free_full (board_outlines, (GDestroyNotify)destroy_object3d);
 }
-
-void
-object3d_test_board_outline (void)
-{
-  object3d *board_outline;
-
-  board_outline = object3d_from_board_outline ();
-  object3d_export_to_step (board_outline, "object3d_test.step");
-  destroy_object3d (board_outline);
-}
